### ISSUE TITLE
Chronograf nightly Debian package fails to install 

### ISSUE THREAD

**Issue by karel-rehor** [influxdata/chronograf#4519]
Build **NIghtly Build**
Bulidtime **n201809282226**

*Use Case*

On Ubuntu 16.04 try installing the latest nightly.deb package using dpkg tool. 
```
$ sudo dpkg -i chronograf.deb 
dpkg: error processing archive chronograf.deb (--install):
 parsing file '/var/lib/dpkg/tmp.ci/control' near line 2 package 'chronograf':
 error in 'Version' field string 'Nightly~n201809282226-0': version number does not start with digit
Errors were encountered while processing:
 chronograf.deb
```

*Workaround*

1. Unpack the debian package 

```
ar x ../chronograf.deb
```
2. Unpack the file control.tar.gz

```
tar xvzf control.tar.gz
```

3. Edit the control file from this package and add "1.7" at the start of the "Version" string 
e.g.
```
Version: 1.7-Nightly~n201809282226-0
```
4. Repack the file control.tar.gz 
5. Repack the debian package 

```
$ ar r chronograf.deb debian-binary control.tar.gz data.tar.gz 
ar: creating chronograf.deb
```
6. Run `dpkg -i chronograf.deb`

*Selenium* 

The nightly Selenium tests use the `dpkg -i chronograf.deb` command when preparing the test environment.

**Comment by karel-rehor** [comment]
In this morning's build this was still not installable.  I had to repackage it manually.   

```
$ chronograf --version
2018/10/02 10:49:11 Chronograf Nightly~n201810020048 (git: 8b0ae52e0652e34ae871119199e4c33617c351c6)
```

This is a blocker for nightly automated tests.  The test environment preparation scripts rely on the debian package for chronograf installation.

**Comment by chnn** [comment]
Fixed by 0a98597aa8936d59237313e3c252c3eadb68ef8c.
